packages/app-store/routing-forms/pages/reporting/[...appPages].tsx

```diff
@@ -266,6 +266,7 @@ const Reporter = ({ form }: { form: inferSSRProps<typeof getServerSideProps>["fo
 };
 
 export default function ReporterWrapper({
```

Comment:
I think we don't use this reporting page anymore. It is superseded by Insights. We should remove this.


'View Responses' CTA in Routing Form takes to Insights page and not this reporting page.

Reply:
Opened a followup ticket